src/components-nextjs/PageSeo/PageSeo.tsx

```diff
@@ -116,12 +117,20 @@ export function PageSeo({
                 cardType: 'summary',
                 ...twitter,
             }}
-            additionalMetaTags={[{ name: 'twitter:image', content: logoUrl }]}
+            additionalMetaTags={[
+                { name: 'twitter:image', content: logoUrl },
+                site.google_search_console_key
+                    ? {
+                          name: 'google-site-verification',
+                          content: site.google_search_console_key,
+                      }
+                    : undefined,
+            ].filter((tag): tag is Exclude<typeof tag, undefined> => Boolean(tag))}
```

Comment:
How about this?
```suggestion
            additionalMetaTags={[
                { name: 'twitter:image', content: logoUrl },
                ...(
                site.google_search_console_key
                    ? [{
                          name: 'google-site-verification',
                          content: site.google_search_console_key,
                      }]
                    : []
                ),  
            ]}
```

Reply:
i fix the types, so you can do like that
```
 additionalMetaTags={[
        { name: 'twitter:image', content: logoUrl },
        site.google_search_console_key && {
            name: 'google-site-verification',
            content: site.google_search_console_key,
        },
    ].filter((tag): tag is Exclude<typeof tag, string | null> => Boolean(tag))}
```

So when you will need to add mot meta tags you will need less characters to add